cmd/influx/setup.go

```diff
@@ -47,27 +53,37 @@ func setupF(cmd *cobra.Command, args []string) error {
 
 	// check if setup is allowed
 	s := &http.SetupService{
-		Addr:               flags.host,
+		Addr:               flags.Host,
 		InsecureSkipVerify: flags.skipVerify,
 	}
-
 	allowed, err := s.IsOnboarding(context.Background())
 	if err != nil {
 		return fmt.Errorf("failed to determine if instance has been configured: %v", err)
 	}
 	if !allowed {
-		return fmt.Errorf("instance at %q has already been setup", flags.host)
+		return fmt.Errorf("instance at %q has already been setup", flags.Host)
 	}
 
-	dPath, dir, err := defaultTokenPath()
+	dPath, dir, err := defaultConfigPath()
 	if err != nil {
 		return err
 	}
 
+	existingConfigs := make(config.Configs)
 	if _, err := os.Stat(dPath); err == nil {
-		return &platform.Error{
-			Code: platform.EConflict,
-			Msg:  fmt.Sprintf("token already exists at %s", dPath),
+		existingConfigs, _ = config.LocalConfigsSVC{
+			Path: dPath,
+			Dir:  dir,
+		}.ParseConfigs()
+		// ignore the error if found nothing
+		if setupFlags.name == "" {
+			return errors.New("flag name is required if you already have existing configs")
```

Comment:
can we just fall back to default here?

Reply:
most likely, it could error out later in the conflict. I would prefer to make this requirement clear here.

Reply:
this new requirement breaks backwards compatability for anyone who has scripts to automate setup in CI or elsewhere. I don't think this is a viable choice.